resources/poker-testdata.txt

```diff
@@ -0,0 +1,19 @@
+*other test data I used to test matching combination*
```

Comment:
is this file being used anywhere?

Reply:
This just contains the matching combinations test...